### PR TITLE
Fix issue 846, update mknb doc

### PR DESCRIPTION
**Pull request by xuweibj**
#846 
http://10.3.5.4/install/build/html/guides/admin-guides/references/man8/mknb.8.html

### CODE REVIEW

**Comment by whowutwut** [comment]
Hi @xuweibj , the `.rst` man page files are auto generaged.  They should not be changed directly, the man pages are in `.pod` files or in Schema.pm`.   Otherwise the changes will not appear when you run `man mkbn`.  

In this case, the pod file is `xCAT-client/pods/man8/mknb.8.pod`.  You should make the changes for mknb.8.pod, then run the `create_man_pages.py` script to generate the resulting .rst files.  See http://xcat-docs.readthedocs.io/en/stable/developers/guides/code/builds.html#man-pages

**Comment by xuweibj** [comment]
Hi @whowutwut , 
I modified, please help review again. Thanks.

File: xCAT-client/pods/man8/mknb.8.pod [hunk 1]
```diff
@@ -11,21 +11,21 @@ B<mknb> I<arch>
 
 The B<mknb> command is run by xCAT automatically, when xCAT is installed on the management node.
```
Comment (whowutwut) [inline xCAT-client/pods/man8/mknb.8.pod]: Can we remove the comma here on this line?

File: xCAT-client/pods/man8/mknb.8.pod [hunk 2]
```diff
@@ -11,21 +11,21 @@ B<mknb> I<arch>
...
 It creates a network boot root image (used for node discovery, BMC programming, and flashing)
-for the same architecture that the management node is.  So you normally do not need to run the B<mknb>
-command yourself.   
+for the same architecture that the management node is.  So you normally do not need to run the 
+B<mknb> command yourself.   
 
 If you do run B<mknb> to add custom utilities to your boot root image, and you have an xCAT Hierarchical Cluster with service nodes that each have a local /tftpboot directory (site sharedtftp=0), you will also need to copy the generated root image to each service node.
```
Comment (whowutwut) [inline xCAT-client/pods/man8/mknb.8.pod]: Suggest here: 
```
If you make custom changes to the network boot root image, you will need to run B<mknb> 
again to regenerate the diskless image to include your changes.  If you have an xCAT Hierarchical
Cluster with Service Nodes having local /tftpboot directories (site.sharedtftp=0), you will need to 
copy the generated root image to each Service Node.
```

**Comment by whowutwut** [comment]
@xuweibj  Great! looks good, I added some minor comments.

File: xCAT-client/pods/man8/mknb.8.pod [hunk 3]
```diff
@@ -11,21 +11,21 @@ B<mknb> I<arch>
...
 
 
-Presently, only the arch x86_64 is supported.
+Presently, the architectures x86_64 and ppc64 are supported.
```
Comment (whowutwut) [inline xCAT-client/pods/man8/mknb.8.pod]: Here, can we add to the end of this..  ``... supported.  For ppc64le, use the ppc64 architecture. ``

I have gotten many questions about if ppc64le is supported.

**Comment by xuweibj** [comment]
Hi @whowutwut 
Thank you for your comments. I modified, please help review again.

**Comment by whowutwut** [comment]
Looks Great!

**Comment by whowutwut** [comment]
Commit to 2.11 branch: 7c76b554f14b815c3a00bea3abe795243fa8286f

**Pull request merged**